rocketpy/motors/fluid.py

```diff
@@ -12,12 +16,18 @@ class Fluid:
     ----------
     name : str
         Name of the fluid.
-    density : float
-        Density of the fluid in kg/m³.
+    density : int, float, callable, string, array, Function
+        Density of the fluid in kg/m³. If a int or float is given,
+        it is considered a constant in time. A callable, csv file
+        or an list of points can be given to express the density
+        as a function of time. The parameter is used as a ``Function``
+        source.
+
+        See more on :class:`rocketpy.Function` source types.
     """
 
     name: str
-    density: float
+    density: Union[int, float, str, list, Function]
 
```

Comment:
Python functions could also be accepted, right? (for example the lambda you applied in the example)

I would let callable be a valid type for the density parameter instead of just rocketpy.Function

```suggestion
    density: Union[int, float, str, list, callable]

```